src/components/common/ratioInput/RatioInput.test.tsx

```diff
@@ -112,18 +132,26 @@ describe("RatioInput Component", () => {
       />
     );
 
-    expect(screen.getByText("Ratio")).toBeInTheDocument();
+    expect(screen.getByText("Numerator")).toBeInTheDocument();
+    expect(screen.getByText("Denominator")).toBeInTheDocument();
     expect(screen.getAllByPlaceholderText("value").length).toBe(2);
     expect(screen.getAllByPlaceholderText("unit").length).toBe(2);
 
-    const valueInput = screen.getAllByTestId(
-      "quantity-value-input-Ratio"
+    const valueInputNumerator = screen.getAllByTestId(
+      `quantity-value-input-Numerator`
```

Comment:
not sure why the consistent lowercase style was not followed- `quantity-value-input-Numerator` vs `quantity-value-input-numerator`. Also template literal her as well

Reply:
Addressed